pkg/operator/ceph/object/rgw.go

```diff
@@ -145,6 +146,16 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string)
 			}
 		}
 
+		// Disable multisite sync traffic
+		if c.store.Spec.Gateway.DisableMultisiteSyncTraffic {
+			monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
+			who := generateCephXUser(rgwConfig.ResourceName)
+			err := monStore.Set(who, "rgw_run_sync_thread", "false")
+			if err != nil {
+				return errors.Wrapf(err, "failed to set rgw_run_sync_thread to false on %q", who)
+			}
+		}
```

Comment:
Thanks for pointing this out! I changed the option from a bool to a pointer so that I can detect whether the option has been specified by the users. To avoid any override of user-defined values via ceph commands applied from the toolbox, I propose to do nothing if the option is not set.